web/src/styles/sections/nav-bar.scss

```diff
@@ -28,14 +28,31 @@ $icon-offset: 45px;
   z-index: 5;
 }
 
+.Mui-focusVisible.nav-menu-item-container {
+  background-color: $transparent;
+}
+
+.Mui-focusVisible.nav-menu-item-container.bg-primary-extra-light-mui-selected-override {
+  background-color: $primary-extra-light;
+}
+
+.Mui-focusVisible.profile-non-selected {
+  background-color: #fbfdf9;
+}
+
 .nav-menu-item-container {
   min-height: 47px;
 }
 
-.profile-default-background-color {
+.profile-non-selected {
```

Comment:
I don't love the naming convention `profile-non-selected`. Is there a better name for this than just saying "non-selected"?

Reply:
So only the profile section (dashboard and profile buttons) has this slight green background other sections like the login / logout buttons / register / myNJ buttons don't so I like how profile reffers to a specific section at a glance

Reply:
It's not the `profile` piece I'm concerned about. `non-selected` feels non-specific and in my opinion breaks specific naming conventions we're using elsewhere.

Reply:
I would expect to see something like `profile-button` and then `profile-button-selected`. This just feels a little backwards to me.

Reply:
Ah, gotcha I misread it. 

I had it originally as `profile-default-background-color` would that be prefferable?

Reply:
I like that more. I'd probably argue for something like `profile-menu-item`. I don't really like the idea of using a CSS property as a class name. It just feels like a code smell to me.

Reply:
Sure, I was imagining it might be clearer given that most of our classes are USWDS so they are descriptive of what they do and its kinda nice to not have to go into an scss file in some cases